LabExT/Movement/Calibration.py

```diff
@@ -311,3 +311,45 @@ def determine_state(self, skip_connection=False) -> None:
             return
 
         self._state = State.FULLY_CALIBRATED
+
+    #
+    #   Position method
+    #
+
+    @assert_minimum_state_for_coordinate_system(
+        stage_coordinate_system=State.CONNECTED,
+        chip_coordinate_system=State.SINGLE_POINT_FIXED)
```

Comment:
Do I get it right that if this function is called within "stage_coordinate" context manager, then you don't care what the chip_coordinate_system state is, right?

Reply:
Yes, the state is selected depending on the current coordinate system.